Plugin/Magento/Eav/Model/Entity/AttributePlugin.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace JustBetter\OptimizeFlatTables\Plugin\Magento\Eav\Model\Entity;
+
+use Magento\Catalog\Model\ResourceModel\Attribute;
+use Magento\Eav\Model\Entity\Attribute\AbstractAttribute;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Zend_Db_Expr as Zend_Db_Expr;
```

Comment:
The alias is the same as the namespace. Is an alias even necessary then?
```suggestion
use Zend_Db_Expr;
```